plopfile.js

```diff
@@ -50,13 +50,11 @@ module.exports = function (plop) {
         const indexFile = `${plop.getDestBasePath()}/packages/components/src/index.ts`;
 
         if (fs.existsSync(indexFile)) {
-          const nameRegex = /default as ([^\s]*)/;
```

Comment:
I think plop was broken on `main` (although I didn't test that). Noticed while doing some upgrades that it's broken. Probably broke when we started exporting types from index.ts.